examples/app.py

```diff
@@ -62,7 +62,7 @@
         cd static
         npm install
         cd ..
-        npm install -g node-sass clean-css requirejs uglify-js
+        npm install -g node-sass@3.8.0 clean-css@3.4.24 requirejs uglify-js
```

Comment:
@lnielsen Yes, I got into troubles with `webassets`, which may take unknown time to release hte fix, so I'm rolling back to pinning the dependencies.

@martinkoehler Please stay tuned, once I sort this out, I'll update also the Invenio demo site.